flathub.json

```diff
@@ -1,3 +1,2 @@
 {
-  "automerge-flathubbot-prs": true
```

Comment:
Any reason why this is being removed?

Reply:
> ### flathub-json-automerge-enabled
> Exceptions allowed: Only for verified applications and extra-data apps if upstream rotates/deletes sources
>
> The flathub.json file has automerge-flathubbot-prs property enabled. This is no longer allowed by default.
-- [Flatpak Builder Lint Docs](https://docs.flathub.org/docs/for-app-authors/linter/#flathub-json-automerge-enabled)

Appears to be something that was changed. There are questions why it isn't working anymore when googling.

Reply:
Oh, if you mean why they changed it? No idea.

Reply:
Oh no, I was asking why you removed it, but you answered my question :)